src/components/ScholarshipAmountField.jsx

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
 import {
-  FormLabel,
-  Radio,
-  RadioGroup,
-  FormControl,
-  FormControlLabel,
+  InputLabel,
+  Grid,
+  Select,
+  MenuItem,
   FormHelperText,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
+import RemoveIcon from '@material-ui/icons/Remove';
 import PropTypes from 'prop-types';
 import AmountType from '../types/AmountType';
 import AmountTextField from './AmountTextField';
 
-const useStyles = makeStyles({
-  amountFieldStyle: {
-    display: 'flex',
-    alignItems: 'center',
+const useStyles = makeStyles((theme) => ({
+  amountSelect: {
+    minWidth: 150,
   },
-});
+  dash: {
+    marginTop: theme.spacing(2),
+  },
+  inputLabel: {
+    marginBottom: theme.spacing(1),
+  },
```

Comment:
Interesting, elsewhere the property `labelStyle` is passed around and this is defined in the `ScholarshipForm` component. I suggest being consistent and either passing the style around everywhere or just assuming a default spacing that'll work well anywhere (e.g., instead of a bottom margin of 2 a top/bottom margin of 1).

Reply:
For now I passed in the style through props. I am wondering if we should just include that style in the different components instead of passing it, since it's just the styling for the label, I don't think it will change.